src/fieri/app/models/foodcritic_worker.rb

```diff
@@ -23,12 +23,26 @@ def make_post(params, feedback, status)
       fieri_key: ENV['FIERI_KEY'],
       cookbook_name: params['cookbook_name'],
       cookbook_version: params['cookbook_version'],
-      foodcritic_feedback: feedback,
+      foodcritic_feedback: format_feedback(feedback, status),
       foodcritic_failure: status
     )
     unless response.is_a? Net::HTTPSuccess
       error_msg = "Unable to POST Foodcritic Evaluation of #{params[cookbook_name]} " + response.message
       raise error_msg
     end
   end
+
+  private
+
+  def foodcritic_info
+    "Run with Foodcritic Version #{FoodCritic::VERSION} with tags #{ENV['FIERI_FOODCRITIC_TAGS']}"
```

Comment:
We ought to include `ENV['FIERI_FOODCRITIC_FAIL_TAGS']` too because that's the list of categories run against (along with the `FIERI_FOODCRITIC_TAGS` which default to some we turn off).